index.js

```diff
@@ -12,11 +12,18 @@ module.exports = (
     columns,
     ...incrementalConfig,
   }).query(
```

Comment:
I think we should try to be backwards compatible here, and I think that's fairly simple. If the `hash` argument is missing, we just revert to the original logic, something like:
```suggestion
  }).query(
  !!hash ?
  <your new code>
  :
  <the old code>;
```

Reply:
For sure! I'll get that added. TYSM for the quick review!

Reply:
Just added this and ran it both ways with test data